### ISSUE TITLE
training with only map head failed

### ISSUE THREAD

**Issue by ASONG0506** [swc-17/SparseDrive#32]
Hello, during my training process, I only set up the training for the map head module and found that the loss value stopped decreasing after dropping to around 6. I am using 4*3090 GPUs with a batch size of 64 and an initial learning rate (lr_init) of 4e-4. Attached is my training log. Could you help me identify the reason for this?
[20240823_155044.log](https://github.com/user-attachments/files/16757339/20240823_155044.log)

after that , I set lr_init=1e-4 instead and here is the result, same result as above
[20240826_183830.log](https://github.com/user-attachments/files/16757304/20240826_183830.log)

and this is my clustered map anchor and det anchor visualizing reuslt, is that normal?
![map_anchor_100](https://github.com/user-attachments/assets/93bffaf1-acd5-4909-b5cd-3fc0394105ee)
![det_anchor_900](https://github.com/user-attachments/assets/a7934966-1985-4776-b37e-9bef4f2de09b)

**Comment by swc-17** [comment]
The visualization results are normal. Can you give a detailed description of your modification on the config file and codes?

**Comment by ASONG0506** [comment]
I didn't change the code actually, and I only made modifications to the following part of the stage1 config file:
`task_config = dict(
    with_det=True,
    with_map=True,
    with_motion_plan=False,
)`
Changed it to:
`task_config = dict(
    with_det=False,
    with_map=True,
    with_motion_plan=False,
)`
This means I modified the with_det option, disabling the detection training part. The learning rate was changed from 4e-4 to 1e-4. Other aspects remained largely unchanged.

**Comment by ASONG0506** [comment]
When I turn on the training of obj detection, the loss of mapping module looks like normal

**Comment by swc-17** [comment]
This is interesting, we previously trained the map-only model and it can converge normally.

**Comment by ASONG0506** [comment]
@swc-17 
I add some auxiliary loss, now it looks like the training convergence is normal

**Comment by shenyinzhe** [comment]
> [@swc-17](https://github.com/swc-17) I add some auxiliary loss, now it looks like the training convergence is normal

I also encountered this problem. My map loss is stuck at ~0.36 and does not decrease anymore, which is exactly the case in your provided logs. 
Could you share your solution? Thanks!

**Comment by nicolas-schischka** [comment]
Hi @ASONG0506, I am having the same issue. Could you share which auxiliary loss you used? Thank you very much!
